apps/frontend/app/(client)/(main)/course/[courseId]/_components/CourseInfoBox.tsx

```diff
@@ -0,0 +1,78 @@
+'use client'
+
+import Calendar from '@/public/icons/calendar-default.svg'
+import Person from '@/public/icons/person.svg'
+import Image from 'next/image'
+import { useSearchParams } from 'next/navigation'
+import { useState, useEffect } from 'react'
+
+export function CourseInfoBox() {
+  const searchParams = useSearchParams()
+  const courseId = searchParams.get('courseId')
+  const [courseCode, setcourseCode] = useState<string>()
+  const [courseName, setcourseName] = useState<string>()
+  const [courseSemester, setcourseSemester] = useState<string>()
+  const [profName, setprofName] = useState<string>()
```

Comment:
1. camelCase로 작성해주세요!
2. 상태의 타입이 객체가 아닌 경우 타입추론을 활용해주세요!
```suggestion
  const [courseCode, setCourseCode] = useState()
  const [courseName, setCourseName] = useState()
  const [courseSemester, setCourseSemester] = useState()
  const [profName, setProfName] = useState()
```